index.html

```diff
@@ -12,206 +12,271 @@
-      </p>
-
-      <h4 class="h4-connect"> LET'S CONNECT</h4>
-      <!--End of Welcome section-->
-
-      <!-- Social Media  Welcome section-->
-      <div class="icons-container">
-        <ul class="social-media-icons">
+  <section class="main-container">
+    <header class="main-header">
+      <nav class="nav">
+        <a href="#" class="logo">My Logo</a>
+        <img src="./media/Union.png" alt="Union" width="30px">
+        <ul>
           <li>
-            <a href="#">
-              <img src="media/twitter.svg" alt="twitter" />
-
-            </a>
+            <a href="#">Portfolio</a>
           </li>
           <li>
-            <a href="#">
-              <img src="media/Linkedin-icon.svg" alt="linkedin" />
-            </a>
+            <a href="#">About</a>
           </li>
           <li>
-            <a href="#">
-              <img src="media/medium.svg" alt="medium" />
-            </a>
-          </li>
-          <li>
-            <a href="#">
-              <img src="media/github.svg" alt="github" />
-            </a>
-          </li>
-          <li>
-            <a href="#">
-              <img src="media/angelist.svg" alt="angellist" />
-            </a>
+            <a href="#">Contact</a>
           </li>
         </ul>
+      </nav>
+
+    </header>
+    <!--Welcome section-->
+
+
+    <main class="main-section">
+
+      <div class="main-title">
+        <h1>I'm Gilbert,<br> Glad to See You!</h1>
+
+        <p>
+          I'm a software developer! I can help you build a product , feature or website Look through some of my work and
+          experience! If you like what you see and have a project you need coded, don't hestiate to contact me.
+        </p>
+
+        <div class="lets-connect"> LET'S CONNECT</div>
+        <!--End of Welcome section-->
+
+        <!-- Social Media  Welcome section-->
+        <div class="icons-container">
+          <ul class="social-media-icons">
+            <li>
+              <a href="#">
+                <img src="media/twitter.svg" alt="twitter" />
+
+              </a>
+            </li>
+            <li>
+              <a href="#">
+                <img src="media/Linkedin-icon.svg" alt="linkedin" />
+              </a>
+            </li>
+            <li>
+              <a href="#">
+                <img src="media/medium.svg" alt="medium" />
+              </a>
+            </li>
+            <li>
+              <a href="#">
+                <img src="media/github.svg" alt="github" />
+              </a>
+            </li>
+            <li>
+              <a href="#">
+                <img src="media/angelist.svg" alt="angellist" />
+              </a>
+            </li>
+          </ul>
+        </div>
       </div>
 
       <!--End of Social media Welcome section-->
-    </section>
-  </main>
+    </main>
+  </section>
 
   <!--These are my projects-->
 
-  <section class="Work-projects">
+  <section class="cards">
```

Comment:
- [ ] As requested by the [previous reviewer](https://github.com/Mutai-Gilbert/Responsive-Personal-Porfolio/pull/4#discussion_r989480162), kindly ensure that the margins/ padding-right and left of the individual cards correspond to the design that is given in Figma. Note that you are not required to produce a very exact copy but a very similar design to the provided one.

**This is how I am seeing your design:**

![image](https://user-images.githubusercontent.com/86318284/194525923-69f4865d-f883-4fad-bf2c-12b2d6be856b.png)

**This is the desired design:**

![image](https://user-images.githubusercontent.com/86318284/194526051-d2bf9c99-09ec-4dc0-87b2-022ead29c9c4.png)

**Note the spacing top and bottom of each individual project card**